### ISSUE TITLE
GRPO Configure

### ISSUE THREAD

**Issue by zghhui** [wdrink/SimpleAR#20]
Hi, this is a good work😀, I have some other questions:
- Is the reward curve the result of training with 32 GPUs or 8 GPUs?
- What are the batch_size and group_size used in your GRPO training? How do these two parameters affect performance?
- Does the scale of data have an impact on the final performance?

**Comment by wdrink** [comment]
Hi @zghhui, thanks for your interest in our work. We train our model on 8 A100 GPUs for RL training. In terms of specific hyper-parameters, please find them in [config.yaml](https://github.com/wdrink/SimpleAR/blob/main/simpar/configs/config_grpo_clip.yaml). 

Compared to pretraining or SFT, GRPO training is considerably slower and compute-consuming. In practice, a moderate data scale, on the order of tens of thousands of samples, is sufficient and feasible. During our experiments, we randomly sample 11k prompts from SFT data, more details can be found in [this discussion](https://github.com/wdrink/SimpleAR/issues/1).  Our results suggest that the key to successful GRPO training lies in maintaining stable reward growth throughout optimization. Notably, such reward improvement typically correlates with better generation performance, as measured by GenEval metrics.

Feel free to reach out for other questions, we are quite glad to facilitate your reproduction.

**Comment by zghhui** [comment]
Thanks for the quick reply! From the reward curve, could I understand that good performance improvement can be achieved within one epoch?

**Comment by wdrink** [comment]
Yes~

**Comment by zghhui** [comment]
This is cool. Have you explored whether other AR models can achieve performance improvements through RL (e.g., LlamaGen)? This implies whether the model's(Qwen2) intrinsic capabilities have been carefully considered.

**Comment by wdrink** [comment]
We don't, but I believe whether we can see significant performance improvements through RL is highly related to the base models' capability.

**Comment by zghhui** [comment]
Very helpful, thanks again🫡.
